xbmc/pvr/epg/EpgInfoTag.cpp

```diff
@@ -73,8 +73,8 @@ CPVREpgInfoTag::CPVREpgInfoTag(const EPG_TAG& data, int iClientId, const std::sh
   m_iEpgID(iEpgID)
 {
   // firstAired is optional, so check if supported before assigning it
-  if (data.firstAired > 0)
-    m_firstAired = time_t(data.firstAired + CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()->m_iPVRTimeCorrection);
+  if (data.strFirstAired)
```

Comment:
How about `if (data.firstAired && strlen(data.strFirstAired) > 0)`. If someone sets this to empty string then that is also invalid?

Reply:
Yes thats make absolutely sense, with the reworked becomes nowhere a `nullptr` given as string.

Reply:
Thanks for the save on this one, gentlemen.  Much better this way, it would have been really annoying for people to set an empty string and end up with 1/1/1970.